appveyor.yml

```diff
@@ -0,0 +1,16 @@
+version: '{build}'
+
+environment:
+    PATH:           'C:\Ruby200\bin;%PATH%'
+    VERSION_SUFFIX: -netstd

```

Comment:
the -netstd? You [asked for it](https://github.com/FakeItEasy/FakeItEasy/pull/821#discussion_r75471409). It had been specified in the clickly-clicky.
Which makes more sense if it's going to be an environment variable.
I'm happy to put it back.